tests/e2e/cucumber/features/keycloak/groups.feature

```diff
@@ -0,0 +1,79 @@
+Feature: groups management
+  As a user
+  I want to manage all group-related operations using ownCloud Web
+  So that I can ensure all group-related operations work correctly with Keycloak integration
+  # For synchronization-related details, see https://owncloud.dev/services/proxy/#claim-updates
+
+
+  Scenario: keycloak group sync with oCIS
+    Given "Admin" creates following user using API
+      | id    |
+      | Alice |
+      | Brian |
+    And "Alice" creates the following files into personal space using API
+      | pathToFile          | content              |
+      | shareToSales.txt    | Keycloak group share |
+      | shareToSecurity.txt | Keycloak group share |
+
+    When "Admin" logs in
+    And "Admin" opens the "admin-settings" app
+    And "Admin" navigates to the groups management page
+    When "Admin" creates the following groups
+      | id       |
+      | security |
+      | finance  |
+      | sales    |
+    Then "Admin" should see the following group
+      | group            |
+      | security         |
+      | finance          |
+      | keycloak sales   |
+      | keycloak finance |
+
+    When "Admin" navigates to the users management page
+    And "Admin" adds the user "Brian" to the groups "finance,keycloak sales" using the sidebar panel
```

Comment:
should we only check adding user to keycloak group?
```suggestion
    And "Admin" adds the user "Brian" to the groups "keycloak sales" using the sidebar panel
```

Reply:
to test keycloak integration doesn't affect oCIS side group

Reply:
different resoure is shared to different group which looks good. :+1: